markdown/assignments/pool_concept/tasks/swing.md

```diff
@@ -0,0 +1,31 @@
+In den Vorgaben finden Sie die Implementierung für ein TicTacToe-Spiel.
+Ihre Aufgabe ist es, eine grafische Benutzeroberfläche für das Spiel zu entwickeln.
+
+Ihr Fenster soll sich immer in der Mitte des Bildschirms starten und einen Titel besitzen.
+
+Beim Start des Spiels, sollen beide Spieler Ihren Namen eingeben können, nutzen Sie dafür `JTextField`. Stellen Sie sicher, dass nur gültige Eingaben getätigt werden.
+Sind die beiden Namen gültig, erstellen Sie die jeweiligen `Player` und starten Sie ein Spiel `TicTacToe`.
+
+Im Zentrum Ihres Spielfensters soll das Spielfeld angezeigt werden. Nutzen Sie dafür das `GridLayout` und `JButton`.
+Die Buttons repräsentieren dabei die Felder des Spiels. Beim Drücken eines Buttons soll die Methode `TicTacToe#makeMove` aufrufen.
+Überlegen Sie sich ein System, um die Parameter für die Methode richtig zu bestimmen.
```

Comment:
??? Versteht man das, wenn man den Code liest?

Reply:
sollte man.
In kurz:
`makeMove` bekommt eine x und y position übergeben, quasi die Koordinate des Kästchen das man ankreuzt. Wenn ich jetzt den Button in der Mitte anklicke, soll dann natürlich `makeMove(1,1)` aufgerufen werden

Reply:
verstehe. aber "System" klingt irgendwie so groß, komplex. ich denke, der satz in zeile 10 reicht völlig, dieser hier in zeile 11 verwirrt ggf. nur  ...  mich jedenfalls hat er verwirrt :)